.installer/verify_install.sh

```diff
@@ -123,6 +124,15 @@ validate_goenv() (
   fi
 )
 
+validate_jenv() (
+  if [[ ! -s "${HOME}/.jenv" ]]; then
+    fail "Failed to locate jenv directory in \$HOME"
+  else
+    success "jenv directory set"
```

Comment:
checking if latest java has been installed and there's global system version set.